.github/workflows/build-latest.yml

```
@@ -54,11 +54,11 @@ jobs:
       sha: ${{ inputs.sha }}
       arch: arm64
       tag: ${{ inputs.tag-prefix }}latest-arm64
-      xdr_ref: v20.0.2
-      core_ref: v20.1.0
+      xdr_ref: v20.1.2
```

I think there's a typo here.
```suggestion
      xdr_ref: v20.1.0
```

---

Nice catch!